src/test/java/io/kestra/plugin/aws/AbstractLocalStackTest.java

```diff
@@ -0,0 +1,32 @@
+package io.kestra.plugin.aws;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@MicronautTest
+@Testcontainers
+public class AbstractLocalStackTest {
+    public static String LOCALSTACK_VERSION = "localstack/localstack:1.4.0";
+
+    protected static LocalStackContainer localstack;
+
+    @BeforeAll
+    static void startLocalstack() {
+        localstack = new LocalStackContainer(DockerImageName.parse(LOCALSTACK_VERSION))
+            .withServices(LocalStackContainer.Service.S3);
```

Comment:
The service to start is different on each test, you can pass it via an abstract protected method for eg.